docs/command-line-interface/topaz-cli/configuration.mdx

```diff
@@ -14,13 +14,15 @@ The [templates install](templates#installing-a-template) command automatically c
 ```
 Flags:
   -h, --help                         Show context-sensitive help.
-      --no-check                     disable local container running check ($TOPAZ_NO_CHECK)
+  -N, --no-check                     disable local container status check ($TOPAZ_NO_CHECK)
```

Comment:
I always found this option's description a bit vague. Maybe we can take this opportunity and rephrase it.
Something like "execute command without verifying that a local topaz container is running".